docker-for-mac/release-notes.md

```diff
@@ -16,10 +16,23 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop Community 3.0.3
+2020-12-21
+
+> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-mac/)
+
+### Bug fixes and minor changes
+
+- Rollback gprcfuse patches that fixed [docker/for-mac#4957](https://github.com/docker/for-mac/issues/4957) and [docker/for-mac#5124](https://github.com/docker/for-mac/issues/5124). They broke widely used mount configurations. See [docker/for-mac#5157](https://github.com/docker/for-mac/issues/5157).
+
+### Known issues
+
+- Some DNS addresses fail to resolve within containers based on Alpine Linux 3.13. See [docker/for-mac#5020](https://github.com/docker/for-mac/issues/5020).
+
```

Comment:
Add:

- There can be timeouts during `docker-compose up` if there are several services being started. See [docker/for-mac#4957](https://github.com/docker/for-mac/issues/4957) and [docker/for-mac#5124](https://github.com/docker/for-mac/issues/5124).